src/ui/server/helpers/is-insurance-route/index.ts

```diff
@@ -0,0 +1,27 @@
+import { INSURANCE_LOWERCASE } from '../../constants';
+
+/**
+ * isInsuranceRoute
+ * checks if route has originalUrl or if the url contains insurance
+ * if empty or contains insurance, then is an insurance route so returns true
+ * if does not contain insurance, then returns false
+ * @param {String} originalUrl of the page the user was on
+ * @returns {Boolean}
+ */
+const isInsuranceRoute = (originalUrl?: string) => {
+  if (!originalUrl) {
+    return true;
+  }
```

Comment:
Is it safe to assume that if there is no `originalUrl` then it's an insurance route? Should it be a required param?

Reply:
yes.  Only the quote pages and the root pages have originalUrl.  All other insurance pages do not have it so it is null.  Better than adding to every file I think

Reply:
I'm not following, the insurance core page variables has `ORIGINAL_URL` so this would always be defined unless i'm missing something?